nucypher/cli/characters/bob.py

```diff
@@ -4,18 +4,20 @@
 from nucypher.characters.control.emitters import IPCStdoutEmitter
 from nucypher.cli import actions, painting
 from nucypher.cli.config import nucypher_click_config
-from nucypher.cli.types import NETWORK_PORT, EXISTING_READABLE_FILE
+from nucypher.cli.types import NETWORK_PORT, EXISTING_READABLE_FILE, EIP55_CHECKSUM_ADDRESS
 from nucypher.config.characters import BobConfiguration
-from nucypher.config.constants import GLOBAL_DOMAIN
+from nucypher.config.constants import DEFAULT_CONFIG_ROOT
 from nucypher.crypto.powers import DecryptingPower
 
 
 @click.command()
 @click.argument('action')
+@click.option('--pay-with', help="Run with a specified account", type=EIP55_CHECKSUM_ADDRESS)
```

Comment:
This is not a good name for Bob's checksum address, as he doesn't pay (currently). In fact, why would Bob need an ETH account?

Reply:
To pay for challenges?  More of a stub than anything, using the same name as Alice.

Reply:
Follow-up Issue https://github.com/nucypher/nucypher/issues/1051